### ISSUE TITLE
EAC3 & AC3 codecs not appearing on chromcast with google tv

### ISSUE THREAD

**Issue by BParks21** [USBhost/MX_FFmpeg#22]
I have the same version of mx player on both my firestick 4k and my chromecast with google tv along with the same codec 1.49.0. When navigating to HW+ audio codecs AC3 and EAC3 appear on the firestick mx player app but not the chromecast with google tv mx player app. They are the only two options missing. I Don't understand why this is? Idk where else to report this so sorry if this is not the appropriate place to do so.

**Comment by BParks21** [comment]
Update: Some files will output dolby digital on the chromecast but not all the files that output dolby digital on the firestick do the same on the chromecast.

**Comment by tleepa** [comment]
I have the same issue.

Previous version of MX Player was working fine with mx_neon.zip codec 1.44.
The current MX Player version (1.51.8) on Chromecast does not "see" mx_neon.zip codec (1.49) in Downloads directory.

**Comment by aeu99184** [comment]
> I have the same issue.
> 
> Previous version of MX Player was working fine with mx_neon.zip codec 1.44. The current MX Player version (1.51.8) on Chromecast does not "see" mx_neon.zip codec (1.49) in Downloads directory.

I have same issue

**Comment by BParks21** [comment]
> I have the same issue.
> 
> Previous version of MX Player was working fine with mx_neon.zip codec 1.44. The current MX Player version (1.51.8) on Chromecast does not "see" mx_neon.zip codec (1.49) in Downloads directory.

That's a seperate issue guys. I'm talking about when you go into mx player options then go to the decoder settings then go to HW+ audio codecs to look for EAC3 & AC3 codecs. They are missing. Update to MX Player 1.52.5 and the codecs pack will work. I'm using that version right now and the pack is selectable.

**Comment by Kshan006421** [comment]
> I have the same version of mx player on both my firestick 4k and my chromecast with google tv along with the same codec 1.49.0. When navigating to HW+ audio codecs AC3 and EAC3 appear on the firestick mx player app but not the chromecast with google tv mx player app. They are the only two options missing. Because of this dolbly digital works on the firestick and not the chromecast. I cannot get dolby to work when playing files in mx player on the chromcast but it always works with the 4k firestick even though everything is configured identical on the hardware and software level. Even with vlc on the chromecast I am able to geet dolby. I Don't understand why this is? Idk where else to report this so sorry if this is not the appropriate place to do so.

i aslo have the same issue , did your find any solution?

**Comment by BParks21** [comment]
No I don't understand why they do not appear.

**Comment by coderhyme** [comment]
> No I don't understand why they do not appear.

After setting media permission to `Allow all the time`, files appear. 
I hope it works for you too.

**Comment by nikagersonlohman** [comment]
Thanks, setting it to `Allow all the time` for Files and Media in Settings > Apps > MX Player > Permissions menu worked for me!

**Comment by GerardEarley** [comment]
Yes I can confirm that changing the File and Media permission to Access all the time allowed MX Player to immediate see the codec zip file in the downloads folder. I wished I'd tried this months ago. 😁👍

**Comment by Livest1** [comment]
I have mine set to allow all the time and it's still not showing them as available code. Only thing I can logically conclude is that I need an older version of the app instead of the newest one.

**Comment by GerardEarley** [comment]
> I have mine set to allow all the time and it's still not showing them as available code. Only thing I can logically conclude is that I need an older version of the app instead of the newest one.

Maybe put the aio .zip file in the downloads folder rather than an individual codec file? That way you'd avoid using the wrong architecture codec file. Just a thought

**Comment by Livest1** [comment]
> Maybe put the aio .zip file in the downloads folder rather than an individual codec file? That way you'd avoid using the wrong architecture codec file. Just a thought 

- No dice. I guess I'll try a few different versions MX Player Pro and MX Player albeit I think they're equally the same considering how I can't have both on a device at the same time

**Comment by rahulsinha23** [comment]
Use this link to download letest version 1.86.0
https://github.com/rahulsinha23/Mx-player-custom-codec/releases/tag/Mxplayer
